loadgen/loadgen.cc

```diff
@@ -182,6 +182,25 @@ auto SampleDistribution<TestMode::PerformanceOnly>(size_t sample_count,
              auto& gen) mutable { return dist(gen); };
 }
 
+/// \brief SampleDistribution for 3D-UNet SingleStream, for v2.0
+// FIXME: meant for 3D UNet SingleStream only at the moment but the logic should work for others
+// TODO: consolidate the distribution generator after v2.0 
+auto SampleDistributionEqualIssue(size_t sample_count, size_t set_size, std::mt19937* rng) {
+  std::vector<size_t> indices;
+  std::vector<size_t> shuffle_indices;
```

Comment:
nit: maybe:

```
std::vector<size_t> shuffle_indices(set_size);
std::iota(shuffle_indices.begin(), shuffle_indices.end(), 0);
```

for clarity?

Reply:
Awesome. :) Pushed the change as suggested. Thanks @nvpohanh !